test/DotNest.Core.SDK.Tests.UI/Recipes/DotNest.Core.SDK.Tests.Default.recipe.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "DotNest.Core.SDK.Tests.Default",
+  "displayName": "TEST: DotNest Core SDK - Default tenant",
+  "description": "Setup recipe for the default tenant of DotNest Tenants SDK, for automated UI test execution.",
+  "author": "Lombiq Technologies",
+  "website": "https://dotnest.com/",
```

Use the same in every recipe. `DotNest.Core.SDK.Setup.recipe` now has a bogus website anyway.

```suggestion
  "website": "https://github.com/Lombiq/DotNest-Core-SDK",
```